### ISSUE TITLE
Export Vault Issue

### ISSUE THREAD

**Issue by BlueBeard63** [KosmosisDire/obsidian-webpage-export#209]
**Describe the bug**
It pops up the error of "Failed to get the render view please try again." when trying to export my selected files and folders.

**Operating System**
Windows 10

**To Reproduce**
1. Click Export Vault
2. Select what you want to export
3. Click Export

**Screenshots**
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/49408282/d83569c5-fa7b-4dca-8249-e3ef8110458d)

### IMPORTANT: Error Log
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/49408282/82c3c71e-0def-4a58-8893-b0646d98dfcd)

**Comment by BlueBeard63** [comment]
By the way, my obsidian and installer are both upto date.

**Comment by winterbeef** [comment]
Same.  I inspect the error console (ctrl-shift-i), and see this:

```
plugin:webpage-html-export:56844 Uncaught (in promise) Error: folderPath must be absolute: .
    at Utils.downloadFiles (plugin:webpage-html-export:56844:13)
    at HTMLExporter.saveExports (plugin:webpage-html-export:57288:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async HTMLExporter.exportFiles (plugin:webpage-html-export:57261:7)
    at async Object.callback (plugin:webpage-html-export:57382:23)
downloadFiles @ plugin:webpage-html-export:56844
saveExports @ plugin:webpage-html-export:57288
processTicksAndRejections @ node:internal/process/task_queues:96
await in processTicksAndRejections (async)
o_ @ app.js:1
t.onChooseItem @ app.js:1
t.onChooseSuggestion @ app.js:1
t.selectSuggestion @ app.js:1
e.useSelectedItem @ app.js:1
e.onSuggestionClick @ app.js:1
s @ enhance.js:1
plugin:webpage-html-export:56570 failed to find render leaf
```

Indeed, near webpage-html-export:56844:
```
  static async downloadFiles(files, rootPath) {
    if (!rootPath.isAbsolute)
      throw new Error("folderPath must be absolute: " + rootPath.asString);
```

**Comment by KosmosisDire** [comment]
This should be fixed in v1.7.2
